schema/2023-05/Form.json

```diff
@@ -0,0 +1,328 @@
+      "type": "string",
+      "enum": ["TSD","EDUCLOUD"]
+    },
+    "collectsPersonalData": {
+      "description": "Does the form collect personal data.",
+      "type": "boolean"
+    },
+    "sensitivePersonalDataCollected": {
+      "description": "Has sensitive data been collected.",
+      "type": "boolean"
+    },
+  },
+  "$defs": {
+    "editor": {
+      "type": "object",
+      "additionalProperties": false,
+	"required": ["username", "type", "personId"],
+      "properties": {
+        "name": {
+          "description": "Name of the editor.",
+          "type": "string"
+        },
+        "email": {
+          "description": "Email of the editor.",
+          "type": "string"
+        },
+        "type": {
+          "description": "User type",
+          "$comment": "See https://github.uio.no/IT-NETTSKJEMA/nettskjema-api/blob/master/src/main/java/no/uio/webapps/nettskjema/api/v3/person/enums/UserType.java",
+          "type": "string"
+        },
+        "personId": {
+          "description": "Unique identifier for this person.",
+          "type": "number"
+        },
+        "username": {
+          "description": "Username of the editor.",
+          "type": "string"
+        }
+      }
+    },
+    "element": {
+      "description": "Elements of the skjema.",
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["sequence", "mandatory"],
+      "properties": {
+        "text": {
+          "description": "The text of the element, e.g. question text or title for a matrix.",
+          "type": "string"
+        },
+        "title": {
+          "description": "Same content as text. Deprecated.",
+          "type": "string"
+        },
+        "sequence": {
+          "description": "Which question number is this.",
+          "type": "number"
+        },
+        "dateFormat": {
+          "$ref": "#/$defs/dateFormat"
+        },
+        "description": {
+          "description": "Description of the element.",
+          "type": "string"
+        },
+        "elementType": {
+          "$ref": "#/$defs/elementType"
+        },
+        "answerOptions": {
+          "description": "Answer options.",
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/answerOption"
+          }
+        },
+        "externalElementId": {
+          "description": "Codebook key for this element",
+          "type": "string"
+        },
+        "subElements": {
+          "type": "array",
+          "items": [{ "$ref": "#/$defs/subElement" }]
+        },
+        "nationalIdNumberType": {
+          "description": "Enum specifying the ID format",
+          "type": "string"
+        },
+        "isAnswerHashed": {
+          "description": "Boolean specifying if the answer should be hashed or not",
+          "type": "boolean"
+        }
+      }
```

Comment:
This is the only change, I reckon I should just copy the folder for a pr, then make the changes in there. If this is too hard to read, I can do that.

Reply:
Do we expect different ways of hashing/encryption? 
If so I'd suggest an enum `pseudonymization` with values `none` and `hashed` (for now). @kajh 

@janlam-uio I agree with Kaj. Since the new field is not required, I don't think we need make a new schema. But as a general comment: for readability it's better to have copying and editing as two separate commits.

Reply:
No, I don't think we will support other forms of hashing/encryption.